lexers/autohotkey.lua

```diff
@@ -0,0 +1,665 @@
+-- AutoHotkey LPeg lexer.
+
+local lexer = lexer
+local P, S = lpeg.P, lpeg.S
+
+local lex = lexer.new(...)
+
+-- Keywords.
+lex:add_rule('keyword', lex:tag(lexer.KEYWORD, lex:word_match(lexer.KEYWORD, true)))
+
+-- Variables.
+lex:add_rule('variable_builtin', lex:tag(lexer.VARIABLE_BUILTIN, 'A_' * lex:word_match(lexer.VARIABLE_BUILTIN, true)))
+
+-- Constants.
+lex:add_rule('constant', lex:tag(lexer.CONSTANT_BUILTIN, S('fF') * lexer.digit * (lexer.digit)^-1
+    + lex:word_match(lexer.CONSTANT_BUILTIN, true)))
+
+-- Functions.
+lex:add_rule('function_builtin',
+  lex:tag(lexer.FUNCTION_BUILTIN, lex:word_match(lexer.FUNCTION_BUILTIN, true)) *
+    #(lexer.space^0 * P('(')^-1))
```

Comment:
I made a comment in your AutoIt lexer about the `#(lexer.space^0 * P('(')^-1)` component, and the same idea applies.

Reply:
Done, LMK if this is okay.